### PR TITLE
Move control track related content in a separate document

### LINKED ISSUE #456: Remove mentions of Control Track in the main spec
So far there is no player (not even VLC with DVD menus) that can handle Control tracks. There is also no tool to create them.

This can be added in a separate document with its own RFC. We just need to keep the value in the `TrackEntry\TrackType` as reserved.

We can probably do the same with `complex` (mixed DV), `logo` (unused AFAIK). Not sure about `metadata`.

### PR DESCRIPTION
**Pull request by robUx4**
The new document is in a dirty state and we may never use it. But it's a way to clean the Control Track reference (an under specified and unused feature) out of the main document.

Fixes #456

### CODE REVIEW

**Comment by hubblec4** [comment]
> move ChapterTrack and ChapterTrackUID in the Control Tracks spec
Is this really only for Control Tracks?

A good player could read the ChapterTrack and uses only the listed tracks (ordered chapters are required).

**Comment by robUx4** [comment]
`ChapterTrack` is useless without a `ChapterTrackUID`, because it means "all tracks", just as if it doesn't exist.

It's true that the `ChapterTrackUID` spec currently says
> In the absence of a control track, choosing this chapter will select the listed Tracks and deselect unlisted tracks.

But just like control track it's not currently used so I'd rather move it out for now than delaying the specifications for something noone has ever used. Neither vlc nor libavformat support these elements.

**Comment by robUx4** [comment]
BTW, the specs contained as many features as we could think of, even not correctly/properly defined because the idea was to have as much "prior art" as possible. So if anyone tries to patent a new feature they can't because we already thought of it. Even the chapter codecs fall in this category except it got to a proof of concept in VLC.

We don't need these extra parts in the v1 to v4 specs which is meant to specify what is currently used in Matroska.

File: chapters.md [hunk 1]
```diff
@@ -339,8 +320,7 @@ of them contain another splitting.
           <ChapLanguage>eng</ChapLanguage>
         </ChapterDisplay>
         <ChapterFlagHidden>0</ChapterFlagHidden>
-        <ChapterFlagEnabled>1</ChapterFlagEnabled>
-      </ChapterAtom>
+        </ChapterAtom>
```
Comment (mbunkus) [inline chapters.md:323]: This change in indentation is wrong.

File: chapters.md [hunk 2]
```diff
@@ -350,8 +330,7 @@ of them contain another splitting.
...
```
Comment (mbunkus) [inline chapters.md:333]: This change in indentation is wrong.

File: chapters.md [hunk 3]
```diff
@@ -361,8 +340,7 @@ of them contain another splitting.
...
```
Comment (mbunkus) [inline chapters.md:343]: This change in indentation is wrong.

File: chapters.md [hunk 4]
```diff
@@ -372,10 +350,8 @@ of them contain another splitting.
...
```
Comment (mbunkus) [inline chapters.md:353]: This change in indentation is wrong.

File: index_control.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+stream = "IETF"
+status = "informational"
+value = "@BUILD_VERSION@"
+
+[[author]]
+initials="S."
+surname="Lhomme"
+fullname="Steve Lhomme"
+ [author.address]
+  email="slhomme@matroska.org"
```
Comment (mbunkus) [inline index_control.md:22]: inconsistent indentation (should probably be two spaces for both lines)

File: index_control.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+[[author]]
+initials="M."
+surname="Bunkus"
+fullname="Moritz Bunkus"
+  [author.address]
+ email="moritz@bunkus.org"
```
Comment (mbunkus) [inline index_control.md:29]: inconsistent indentation (should probably be two spaces for both lines)

File: chapters.md [hunk 7]
```diff
@@ -24,35 +24,21 @@ set to `true` and the second with `ChapterFlagHidden` either set to `false` or n
...
 
 
 #### EditionFlagDefault
 
 It is **RECOMMENDED** that no more than one `Edition` have an `EditionFlagDefault Flag`
-set to `true`. The first `Edition` with both the `EditionFlagDefault Flag` set to `true`
-and the `EditionFlagHidden Flag` set to `false` is the Default Edition. When all
-`EditionFlagDefault Flags` are set to `false`, then the first `Edition` with the
-`EditionFlagHidden Flag` set to `false` is the Default Edition. The Default Edition
+set to `true`. The first `Edition` with the `EditionFlagDefault Flag` set to `true`
```
Comment (mbunkus) [inline chapters.md:38]: Is the trailing space here intentional? Trailing spaces are significant for Markdown (two spaces == forced line break),
though this instance looks more like an accident.

**Review by mbunkus — CHANGES_REQUESTED** [review]

**Comment by robUx4** [comment]
Rebased after the #342 merge which had more text based on now-moved elements. As a result some tables and text have been simplified.

**Comment by robUx4** [comment]
Rebase after #469 merged.

**Pull request merged**